kokoro/config/test/image_lists.gcl

```diff
@@ -134,7 +134,7 @@ sles15_x86_64 = _distro {
     'opensuse-leap',
     'opensuse-leap-15-4',
   ]
-  presubmit = ['sles-15']
+  presubmit = release
```

Comment:
Is this change intended to be submitted?

Reply:
oops, sent https://github.com/GoogleCloudPlatform/ops-agent/pull/1458.